src/models/index.js

```diff
@@ -2,13 +2,12 @@
 import { readdirSync } from 'fs';
 import { basename as _basename, join } from 'path';
 import Sequelize from 'sequelize';
+import configEnv from '../config';
 
 const basename = _basename(__filename);
-const env = process.env.NODE_ENV || 'development';
 // eslint-disable-next-line import/no-dynamic-require
```

Comment:
Great work here! We don't need to this line anymore.

Reply:
Oh yes.Corrected now